### ISSUE TITLE
EPUBCheck not reporting span elements on block level

### ISSUE THREAD

**Issue by tofi86** [w3c/epubcheck#1070]
A pagina EPUB-Checker user is reporting and asking why EPUBCheck is not reporting isolated span elements on block level like this:

```html
<body>
  <h1>
    <div>div test</div>
  </h1>
  <span>span test</span>
</body>
```

he would expect an error for the span element because it should be wrapped in a block element like div or p.

This was the cae for EPUB 2 validation (XHTML):

```
ERROR (RSC-005) [...] Error while parsing file: element "div" not allowed here; [...]
ERROR (RSC-005) [...] Error while parsing file: element "span" not allowed here; [...]
```

However, with current EPUBCheck only the first error for the div element is reported.

Why is the span error missing in EPUB 3?

Is it maybe a html 5 schema issue?

Thanks for your help,
Tobias

**Comment by rdeltour** [comment]
Yes, this behaves as expected, i.e. as defined in HTML:

- the content model of the [`body`](https://html.spec.whatwg.org/#the-body-element) element is [Flow Content](https://html.spec.whatwg.org/#flow-content-2) (which basically includes most of the HTML elements used for content).
- the content model of [heading elements](https://html.spec.whatwg.org/#the-h1,-h2,-h3,-h4,-h5,-and-h6-elements) is [Phrasing Content](https://html.spec.whatwg.org/#phrasing-content-2) (which is basically the elements allowed in a paragraph).

**Comment by rdeltour** [comment]
Closing as `invalid`, as we're following the spec here 😊

**Comment by tofi86** [comment]
Alright, thanks for clarifying this! 😊
